### ISSUE TITLE
Vue 3 support

### ISSUE THREAD

**Issue by hypericon** [lusaxweb/vuesax#961]
Hello, I would love to use this UI framework as it's amazing!
but I also would love to use while migrating to vue 3.

do you plan any migration / support for Vue3 and, if, so, timings?

tyvm!

**Comment by mentamarindos** [comment]
So sad this project is so underrated, not even the creators has interest on it anymore :disappointed:

**Comment by Stephenjoel2k** [comment]
Yeah, really unfortunate. Had a huuge potential. Looks kinda dead with the lack of support.
